src/ralph/assets/subscribers.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+import logging
+import pyhermes
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from ralph.assets.models.assets import Environment, Service, ServiceEnvironment
+
+logger = logging.getLogger(__name__)
+
+
+def _update_service_owners(service, business_owners, technical_owners):
+    """
+    Update service owners
```

Comment:
What's the point for such docstrings, if they're just a copy of funciton's name..? :wink: